.circleci/config.yml

```diff
@@ -4,205 +4,205 @@ defaults: &defaults
   working_directory: ~/dd-trace-java
   resource_class: xlarge
   docker:
-  - image: &default_container datadog/dd-trace-java-docker-build:latest
+    - image: &default_container datadog/dd-trace-java-docker-build:latest
 
 cache_keys: &cache_keys
   # Reset the cache approx every release
   keys:
-  - trace-java-{{ checksum "trace-java.gradle" }}-{{ .Branch }}-{{ .Revision }}
-  - trace-java-{{ checksum "trace-java.gradle" }}-{{ .Branch }}
-  - trace-java-{{ checksum "trace-java.gradle" }}
+    - trace-java-{{ checksum "trace-java.gradle" }}-{{ .Branch }}-{{ .Revision }}
+    - trace-java-{{ checksum "trace-java.gradle" }}-{{ .Branch }}
+    - trace-java-{{ checksum "trace-java.gradle" }}
 
 jobs:
   build:
     <<: *defaults
 
     steps:
-    - checkout
+      - checkout
 
-    - restore_cache:
-        <<: *cache_keys
+      - restore_cache:
+          <<: *cache_keys
 
-    - run:
-        name: Build Project
-        command: GRADLE_OPTS="-Dorg.gradle.jvmargs='-Xmx1G -Xms64M' -Dio.opentelemetry.auto.forkedMaxHeapSize=1G -Dio.opentelemetry.auto.forkedMinHeapSize=64M" ./gradlew clean :java-agent:shadowJar compileTestGroovy compileLatestDepTestGroovy compileTestScala compileLatestDepTestScala compileTestJava compileLatestDepTestJava --build-cache --parallel --stacktrace --no-daemon --max-workers=8
+      - run:
+          name: Build Project
+          command: GRADLE_OPTS="-Dorg.gradle.jvmargs='-Xmx1G -Xms64M' -Dota.forkedMaxHeapSize=1G -Dota.forkedMinHeapSize=64M" ./gradlew clean :java-agent:shadowJar compileTestGroovy compileLatestDepTestGroovy compileTestScala compileLatestDepTestScala compileTestJava compileLatestDepTestJava --build-cache --parallel --stacktrace --no-daemon --max-workers=8
```

Comment:
I think this needs to match up with https://github.com/open-telemetry/opentelemetry-auto-instr-java/blob/master/trace-java.gradle#L47-L48.

Reply:
nice catch! this is fixed now